torch/_decomp/decompositions.py

```diff
@@ -2092,6 +2092,70 @@ def params_hiddens(params, hiddens, i, bidirectional):
     return cur_params, cur_hidden, bidir_params, bidir_hidden
 
 
+def update_hidden_for_packed(cur_hidden, last_batch_size, batch_size, hiddens):
+    assert last_batch_size > batch_size
+    hiddens.append(cur_hidden.narrow(0, batch_size, last_batch_size - batch_size))
+    return cur_hidden.narrow(0, 0, batch_size)
+
+
+def update_hidden_for_packed_reverse(
+    cur_hidden, last_batch_size, batch_size, inp_hidden
+):
+    assert last_batch_size < batch_size
+    return torch.concat(
+        (
+            cur_hidden,
+            inp_hidden.narrow(0, last_batch_size, batch_size - last_batch_size),
+        )
+    )
```

Comment:
I don't see these two helper functions being used, you probably meant to replace the things in `one_layer_rnn_data` with it.

Reply:
Good catch! Fixed so that they're being used